src/Skoruba.IdentityServer4.STS.Identity/appsettings.json

```diff
@@ -1,5 +1,39 @@
 ﻿{
-	"ConnectionStrings": {
-		"AdminConnection": "Server=(localdb)\\mssqllocaldb;Database=IdentityServer4Admin;Trusted_Connection=True;MultipleActiveResultSets=true"
-	}
+    "ConnectionStrings": {
+        "AdminConnection": "Server=(localdb)\\mssqllocaldb;Database=IdentityServer4Admin;Trusted_Connection=True;MultipleActiveResultSets=true"
+    },
+    "Serilog": {
+        "MinimumLevel": "Error",
+        "WriteTo": [
+            {
+                "Name": "File",
+                "Args": {
+                    "path": "Log\\skoruba_admin.txt",
+                    "rollingInterval": "Day"
+                }
+            },
+            {
+                "Name": "MSSqlServer",
+                "Args": {
+                    "connectionString": "Server=(localdb)\\mssqllocaldb;Database=IdentityServer4Admin;Trusted_Connection=True;MultipleActiveResultSets=true",
+                    "tableName": "Log",
+                    "columnOptionsSection": {
+                        "addStandardColumns": [ "LogEvent" ],
+                        "removeStandardColumns": [ "Properties" ]
+                    }
+                }
+            }
+        ]
+    },
+    "CertificateConfiguration": {
+        "UseTemporaryKey": false,
```

Comment:
Set this to true by default - this is better for someone who want's to try Admin out without having to configure certificates at the very beggining.